src/applications/facility-locator/components/LocationHours.jsx

```diff
@@ -1,107 +1,76 @@
-        Sentry.captureMessage('API location hours data is malformed');
-      });
-
-      return '';
-    }
-
-    return hours.map(time => time.format('h:mmA')).join(' - ');
-  }
-
-  formatLocationHours(hours) {
-    return Object.keys(hours).reduce((accum, key) => {
-      if (hours[key] === '-' || hours[key] === 'Closed') {
-        return { ...accum, [key]: 'Closed' };
-      }
+const LocationHours = ({ location }) => {
+  // Derive the hours for each day.
+  const hoursInfo = get(location, 'attributes.hours');
+  const sunday = get(hoursInfo, 'sunday', 'N/A');
+  const monday = get(hoursInfo, 'monday', 'N/A');
+  const tuesday = get(hoursInfo, 'tuesday', 'N/A');
+  const wednesday = get(hoursInfo, 'wednesday', 'N/A');
+  const thursday = get(hoursInfo, 'thursday', 'N/A');
+  const friday = get(hoursInfo, 'friday', 'N/A');
+  const saturday = get(hoursInfo, 'saturday', 'N/A');
+
+  return (
+    <div>
+      <h4 className="highlight">Hours of Operation</h4>
+
+      {/* Sunday */}
+      <div className="row">
+        <div className="small-6 columns">Sunday:</div>
+        <div className="small-6 columns">{sunday}</div>
+      </div>
 
-      if (!this.formatTimeRange(hours[key])) {
-        return { ...accum };
-      }
+      {/* Monday */}
+      <div className="row">
+        <div className="small-6 columns">Monday:</div>
+        <div className="small-6 columns">{monday}</div>
+      </div>
 
-      return { ...accum, [key]: this.formatTimeRange(hours[key]) };
-    }, {});
-  }
+      {/* Tuesday */}
+      <div className="row">
+        <div className="small-6 columns">Tuesday:</div>
+        <div className="small-6 columns">{tuesday}</div>
+      </div>
 
-  isLocationDataValid(location) {
-    if (!location) {
-      return false;
-    }
+      {/* Wednesday */}
+      <div className="row">
+        <div className="small-6 columns">Wednesday:</div>
+        <div className="small-6 columns">{wednesday}</div>
+      </div>
 
-    const isVetCenter = location.attributes.facilityType === 'vet_center';
+      {/* Thursday */}
+      <div className="row">
+        <div className="small-6 columns">Thursday:</div>
+        <div className="small-6 columns">{thursday}</div>
+      </div>
 
-    if (
-      every(values(location.attributes.hours), hour => !hour) &&
-      !isVetCenter
-    ) {
-      return false;
-    }
+      {/* Friday */}
+      <div className="row">
+        <div className="small-6 columns">Friday:</div>
+        <div className="small-6 columns">{friday}</div>
+      </div>
 
-    return true;
-  }
+      {/* Saturday */}
+      <div className="row">
+        <div className="small-6 columns">Saturday:</div>
+        <div className="small-6 columns">{saturday}</div>
+      </div>
 
-  renderVetCenterContent() {
-    const { location } = this.props;
-    if (location && location.attributes.facilityType === 'vet_center') {
-      return (
+      {get(location, 'attributes.facilityType') === 'vet_center' && (
```

Comment:
I _think_ `vet_center` is a constant somewhere in the FL app directory, to help get rid of that magic string.